src/Function/Definitions.agda

```diff
@@ -28,6 +28,12 @@ open import Level using (_⊔_)
 Congruent : (A → B) → Set (a ⊔ ℓ₁ ⊔ ℓ₂)
 Congruent f = ∀ {x y} → x ≈₁ y →  f x ≈₂ f y
 
+Congruent₂′ : (A → B → A) → Set (a ⊔ b ⊔ ℓ₁ ⊔ ℓ₂)
+Congruent₂′ f = ∀ {x y w z} → x ≈₁ y → w ≈₂ z → f x w ≈₁ f y z
+module _ {c ℓ₃} {C : Set c} (_≈₃_ : Rel C ℓ₃) where
+  Congruent₂ : (A → B → C) → Set (a ⊔ b ⊔ ℓ₁ ⊔ ℓ₂ ⊔ ℓ₃)
+  Congruent₂ f = ∀ {x y w z} → x ≈₁ y → w ≈₂ z → f x w ≈₃ f y z
```

Comment:
These definitions probably already exist in the `Algebra.Module.Definitions.(Left/Right/Bi)` modules? If not, then they should be added there.